capellambse/model/__init__.py

```diff
@@ -324,3 +328,171 @@ def search(
     def by_uuid(self, uuid: str) -> common.GenericElement:
         """Search the entire model for an element with the given UUID."""
         return common.GenericElement.from_model(self, self._loader[uuid])
+
+    # pylint: disable=line-too-long
+    def update_diagram_cache(
+        self,
+        capella_cli: str,
+        image_format: t.Literal["bmp", "gif", "jpg", "png", "svg"] = "svg",
+        create_index: bool = False,
+        force: t.Literal["exe", "img"] | None = None,
+    ) -> None:
+        """Update the diagram cache if one has been specified.
+
+        If a diagram cache has been specified during the load of a
+        Capella model it will be updated when this function is called.
+        The diagram cache will be populated by executing the Capella
+        function "Export representations as images" which is
+        normally accessible via the context menu of an ``.aird`` node in
+        Capella's project explorer. The export of diagrams happens with
+        the help of Capella's command line interface (CLI).
+        The CLI of Capella must be specified by the caller. It is
+        possible to work with a local installation or a Docker image of
+        an individual Capella bundle.
+        At the moment it is supported to run a Docker image using the
+        container system Docker and the ``docker`` executable must be in
+        the ``PATH`` environment variable.
```

Comment:
"run a Docker image using the container system Docker" - well, how else would you run one? I think the second part is redundant here :)

Also, this is explained in depth at the `capella_cli` parameter.

```suggestion
        The CLI of Capella must be specified by the caller; refer to the
        documentation of ``capella_cli`` below for details.
```

Reply:
> "run a Docker image using the container system Docker" - well, how else would you run one? I think the second part is redundant here :)

`docker` is not the only container system, Docker images follow the `OCI` image standard. Therefore there are also alternative solutions, which also implement the `OCI` standard, e.g., [podman](https://podman.io/getting-started/).

Reply:
I'd like to leave this docstring as it is.